components/gc-featured-link/gc-featured-link-fr.html

```diff
@@ -102,99 +83,94 @@ <h2 class="wb-inv">En vedette</h2>
-						<p class="pull-left mrgn-bttm-0"><a class="stretched-link text-white" href="#">[Titre de la promotion]</a></p>
-					</div>
-				</div>
+	<h2>Exemple de lien en vedette avec description sous le lien</h2>
+</div>
+
+<section class="gc-featured-link">
+	<div class="container">
+		<div class="row">
+			<div class="col-xs-12">
+				<h2 class="wb-inv">En vedette</h2>
+				<p><a class="stretched-link" href="#">[Titre de la promotion]</a></p>
+				<p>Description longue [facultatif]</p>
 			</div>
 		</div>
+	</div>
+</section>
 
-		<pre><code>&lt;div class="bg-gctheme opct-90"&gt;
-			&lt;div class="container"&gt;
-				&lt;div class="row"&gt;
-					&lt;div class="mrgn-tp-md mrgn-bttm-md col-xs-12"&gt;
-						&lt;p class="pull-left text-white mrgn-bttm-0"&gt;En vedette&nbsp;:&nbsp;&lt;/p&gt;
-						&lt;p class="pull-left mrgn-bttm-0"&gt;&lt;a class="stretched-link text-white" href="#"&gt;[Titre de la promotion]&lt;/a&gt;&lt;/p&gt;
-					&lt;/div&gt;
-				&lt;/div&gt;
+<div class="container">
+	<h3 id="codeSample3">Code</h3>
+	<pre><code>&lt;section class="gc-featured-link"&gt;
+	&lt;div class="container"&gt;
+		&lt;div class="row"&gt;
+			&lt;div class="col-xs-12"&gt;
+				&lt;h2 class="wb-inv"&gt;Featured&lt;/h2&gt;
+				&lt;p&gt;&lt;a class="stretched-link" href="#"&gt;[Titre de la promotion]&lt;/a&gt;&lt;/p&gt;
+				&lt;p&gt;Description longue [facultatif]&lt;/p&gt;
 			&lt;/div&gt;
 		&lt;/div&gt;
-		</code></pre>
-	</details>
+	&lt;/div&gt;
+&lt;/section&gt;</code></pre>
 
-	<div lang="en">
-	<h2>Evaluation and report</h2>
+	<h2>Évaluations et rapports</h2>
 	<ul>
-		<li>Accessibility pre-assessment #1
+		<li>Pré-évaluation de l'accessibilité #1
 			<ul>
-				<li><a href="./reports/a11y-1-en.html" hreflang="en"><span class="wb-inv">Pre-assessment report #1</span> English report</a></li>
-				<li><a href="./reports/a11y-1-fr.html" hreflang="fr"><span class="wb-inv">Pre-assessment report #1</span> French report</a></li>
+				<li><a href="./reports/a11y-1-en.html" hreflang="en"><span class="wb-inv">Pré-évaluation de l'accessibilité #1</span> Rapport en anglais</a></li>
+				<li><a href="./reports/a11y-1-fr.html" hreflang="fr"><span class="wb-inv">Pré-évaluation de l'accessibilité #1</span> Rapport en français</a></li>
 			</ul>
 		</li>
-		<li>Accessibility assessment #1
+		<li>Évaluation de l'accessibilité #1
 			<ul>
-				<li><a href="./reports/a11y-2-en.html" hreflang="en"><span class="wb-inv">Assessment report #1</span> English report</a></li>
-				<li><a href="./reports/a11y-2-fr.html" hreflang="fr"><span class="wb-inv">Assessment report #1</span> French report</a></li>
+				<li><a href="./reports/a11y-2-en.html" hreflang="en"><span class="wb-inv">Évaluation de l'accessibilité #1</span> Rapport en anglais</a></li>
+				<li><a href="./reports/a11y-2-fr.html" hreflang="fr"><span class="wb-inv">Évaluation de l'accessibilité #1</span> Rapport en français</a></li>
 			</ul>
 		</li>
 	</ul>
```

Comment:
You can remove this section "Évaluations et rapports" because we do have those link via our index.json-ld file.

Can you make the same translation fixes, but for the index.json-ld file? Thanks